profile_library/homewizard/manufacturer.json

```
@@ -0,0 +1,4 @@
+{
+  "name": "HomeWizard",
```

```suggestion
  "name": "homewizard",
```

Please see my reply in #3201, will be corrected in the near future.